### PR TITLE
HSEARCH-2297 Avoid Logger method tracef(String, int) to maintain back…

### PR DESCRIPTION
**Pull request by Sanne**
…wards compatibility

 - https://hibernate.atlassian.net/browse/HSEARCH-2297

### CODE REVIEW

File: orm/src/main/java/org/hibernate/search/query/hibernate/impl/LookupObjectInitializer.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public void initializeObjects(List<EntityInfo> entityInfos, LinkedHashMap<Entity
 		final int maxResults = entityInfos.size();
 		if ( maxResults == 0 ) {
 			if ( traceEnabled ) {
-				log.tracef( "No object to initialize", maxResults );
+				log.tracef( "No object to initialize", (Integer) maxResults );
```
Comment (gsmet) [inline orm/src/main/java/org/hibernate/search/query/hibernate/impl/LookupObjectInitializer.java]: Here we can probably simply remove the parameter.

File: orm/src/main/java/org/hibernate/search/query/hibernate/impl/PersistenceContextObjectInitializer.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ public void initializeObjects(List<EntityInfo> entityInfos,
 
 		if ( numberOfObjectsToInitialize == 0 ) {
 			if ( log.isTraceEnabled() ) {
-				log.tracef( "No object to initialize", numberOfObjectsToInitialize );
+				log.tracef( "No object to initialize", (Integer) numberOfObjectsToInitialize );
```
Comment (gsmet) [inline orm/src/main/java/org/hibernate/search/query/hibernate/impl/PersistenceContextObjectInitializer.java]: Here too.

File: orm/src/main/java/org/hibernate/search/query/hibernate/impl/SecondLevelCacheObjectInitializer.java [hunk 3]
```diff
@@ -38,7 +38,7 @@ public void initializeObjects(List<EntityInfo> entityInfos, LinkedHashMap<Entity
 		final int maxResults = entityInfos.size();
 		if ( maxResults == 0 ) {
 			if ( traceEnabled ) {
-				log.tracef( "No object to initialize", maxResults );
+				log.tracef( "No object to initialize", (Integer) maxResults );
```
Comment (gsmet) [inline orm/src/main/java/org/hibernate/search/query/hibernate/impl/SecondLevelCacheObjectInitializer.java]: Here too.

Comment (Sanne) [inline orm/src/main/java/org/hibernate/search/query/hibernate/impl/SecondLevelCacheObjectInitializer.java]: Good points :+1: 

Now we need a validation plugin to find mismatches like these ;)

**Comment by Sanne** [comment]
@gsmet removed the redundant parameters

**Comment by gsmet** [comment]
Fixed one you missed and merged! Thanks!

**Pull request closed without merging**